### PR TITLE
Added tmath increment

### PR DESCRIPTION
**Pull request by paulhandy**

### CODE REVIEW

File: tmath/src/increment.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+use trytes::*;
+
+pub trait TrinaryIncr {
```
Comment (th0br0) [inline tmath/src/increment.rs]: Please add documentation.

File: tmath/src/increment.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        let high = usize::max_value();
+        let low = usize::min_value();
+        assert_eq!(bct, vec![(high, high), (high, high), (low, high)]);
+
+        let tbc: Trinary = bct.iter().cloned().collect();
+        assert_eq!(i, tbc);
+    }
+    #[test]
+    fn test_smple_incr() {
```
Comment (th0br0) [inline tmath/src/increment.rs]: typo

File: tmath/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+extern crate iota_trytes as trytes;
```
Comment (th0br0) [inline tmath/src/lib.rs:3]: please add 
`#![no_std]`

**Review by th0br0 — CHANGES_REQUESTED** [review]

File: tmath/src/lib.rs (hunk 3 above)
Comment (paulhandy) [reply tmath/src/lib.rs:3]: done

File: tmath/src/lib.rs [hunk 4]
```diff
@@ -1,3 +1,6 @@
+#![feature(alloc)]
+#![no_std]
 extern crate iota_trytes as trytes;
+extern crate alloc as collections;
```
Comment (th0br0) [inline tmath/src/lib.rs]: Don't do the alias? Just removed that from the other crates as well via #57

**Review by th0br0 — DISMISSED** [review]

File: tmath/src/lib.rs (hunk 4 above)
Comment (paulhandy) [reply tmath/src/lib.rs]: ok

**Review by th0br0 — APPROVED** [review]

**Comment by th0br0** [comment]
@iotaculus r+

**Comment by th0br0** [comment]
@iotaculus r+

**Comment by th0br0** [comment]
@iotaculus r+

**Comment by th0br0** [comment]
@iotaculus r+

**Comment by iotaculus** [comment]
:pushpin: Commit 42f9274 has been approved by `th0br0`

**Comment by iotaculus** [comment]
:hourglass: Testing commit 42f9274031c9d6964c026cf666042d25fff03a55 with merge 52627c36714edf87435aeda85f540e8c30be9d77...

**Comment by iotaculus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/iotaledger/iota.rs/builds/248155844?utm_source=github_status&utm_medium=notification)
Approved by: th0br0
Pushing 52627c36714edf87435aeda85f540e8c30be9d77 to master...

**Pull request merged**